README.md

```diff
@@ -108,16 +119,23 @@ Your Answer:
 
 What are the differences between calling and referencing a function? Please provide examples of each.
 
-```text
-
+```
+When a function is called (or invoked), the code inside will execute. This can happen when an event occurs or when it is called within a block of JS code.
+$("p").on("click", doSomething);
 ```
 ### Question #10
 
 Using the object literal notation, Define an object called student and give it the properties (your attributes) of name, age, and a method sayHello, that outputs "Hi, my name is [your_name]".
 
 Your Answer:
-```js
-// write code here
+```
+var student = {
+    name: "James Graham",
+    age: 28,
+    sayHello: function() {
+        console.log("Hi, my name is "+student.name)
+    }
+}
 ```
```

Comment:
:+1: Although ideally you would be referencing the name using `this.name` in `sayHello`.